src/datepicker/datepicker.ts

```diff
@@ -100,6 +101,8 @@ export interface NgbDatepickerNavigateEvent {
         </div>
       </ng-template>
     </div>
+
+    <ng-container *ngTemplateOutlet="footerTemplate; context: { $implicit: datepicker, datepicker: datepicker }"></ng-container>
```

Comment:
Let's replace this with `<ng-template [ngTemplateOutlet]></ng-template>` form and drop context